apps/desktop-wallet/package.json

```diff
@@ -131,6 +131,7 @@
     "react-confetti": "^6.0.1",
     "react-detect-click-outside": "^1.1.2",
     "react-dom": "^18.2.0",
+    "react-freeze": "^1.0.4",
```

Comment:
It was introduced for 2 reasons:
1. Improve UX of tab navigation
2. Improve performance by "freezing" the hidden tab content